taxi/views.py

```diff
@@ -104,14 +158,14 @@ class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
 
 class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
     model = Driver
-    success_url = reverse_lazy("")
+    success_url = reverse_lazy("taxi:driver-list")
 
 
 @login_required
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
```

Comment:
The `Driver` object is retrieved using the request user's ID, which may not be correct if the function is intended to toggle assignment for any driver. Consider revising this logic if necessary.